go-apps/meep-ams/server/ams.go

```diff
@@ -497,12 +513,13 @@ func getAppInstanceId() (id string, err error) {
 	var appInfo scc.ApplicationInfo
 	appInfo.Id = instanceId
 	appInfo.Name = serviceCategory
+	appInfo.Type_ = "SYSTEM"
 	appInfo.MepName = mepName
-	appInfo.Version = serviceAppVersion
-	appType := scc.SYSTEM_ApplicationType
-	appInfo.Type_ = &appType
-	state := scc.INITIALIZED_ApplicationState
-	appInfo.State = &state
+	if mepName == defaultMepName {
```

Comment:
Comment -- Here, it's fine to use `mep` since we are in ETSI MEC land.

Reply:
Okay.